# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: security/manager/ssl/tests/unit/test_trust_anchors/int-by-ca2.pem.certspec
# Commit: 55e22304ccae
# Full Hash: 55e22304ccaea2af33219077000171821441822c
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2025-08-07 04:23:39
# Description:
#   Bug 1981271 - reset state specific to the built candidate chain root when continuing path building r=nkulatova
#   
#   Any state that NSSCertDBTrustDomain keeps with regard to a built candidate
#   chain (and in particular, the root of that chain), must be reset if path
#   building continues with other potential chains. In particular, this means that
# ==============================================================================

diff -r 2c224e8a6135 -r 55e22304ccae security/manager/ssl/tests/unit/test_trust_anchors/int-by-ca2.pem.certspec
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/security/manager/ssl/tests/unit/test_trust_anchors/int-by-ca2.pem.certspec	Wed Aug 06 18:13:58 2025 +0000
@@ -0,0 +1,4 @@
+issuer:ca2
+subject:int
+extension:basicConstraints:cA,
+extension:keyUsage:keyCertSign,cRLSign
